src/components/TextInputIcon/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Icon from 'react-native-vector-icons/Feather';
+import { Colors } from '../../utils/Colors';
+import { TextInput } from '../TextInput';
+
+export const TextInputIcon = ({ 
+  errorInput = false, 
+  msgError = "",
+  icon, 
+  isPassword, 
+  active, label, updateIcon, ...props }) => {
```

Comment:
Adicionar o tipo de retorno do seu componente.
```suggestion
export const TextInputIcon = ({ 
  errorInput = false, 
  msgError = "",
  icon, 
  isPassword, 
  active, label, updateIcon, ...props }): ReactElement => {
```